### DISCUSSION TITLE
GitVersion fails on windows in Azure Pipelines

### DISCUSSION THREAD

**Discussion by nick5454** [cake-build/cake#3308] [Extension Q&A]
Saturday I started getting this on windows for CakeCore version 38.5

Not sure if anyone has seen this using the tool: #module nuget:?package=Cake.DotNetTool.Module&version=0.4.0

In Windows it fails: '''Executing task: MainVersion
Version Started
Resolved tool to path C:/ProgramData/Chocolatey/bin/GitVersion.exe
Executing: "C:/ProgramData/Chocolatey/bin/GitVersion.exe" -output json -updateassemblyinfo -verbosity Debug
The configuration file was not found at 'D:\a\1\s\build\GitVersion.yml' or 'D:\a\1\s\GitVersion.yml'
An error occurred when executing task 'MainVersion'.'''

In MacOs it works
'''Executing task: MainVersion
Version Started
Could not resolve path for tool "GitVersion.exe" using these directories: /usr/local/microsoft/powershell/7,/usr/local/opt/pipx_bin,/Users/runner/.cargo/bin,/usr/local/lib/ruby/gems/2.7.0/bin,/usr/local/opt/ruby@2.7/bin,/usr/local/opt/curl/bin,/usr/local/bin,/usr/local/sbin,/Users/runner/bin,/Users/runner/.yarn/bin,/Users/runner/Library/Android/sdk/tools,/Users/runner/Library/Android/sdk/platform-tools,/Users/runner/Library/Android/sdk/ndk-bundle,/Library/Frameworks/Mono.framework/Versions/Current/Commands,/usr/bin,/bin,/usr/sbin,/sbin,/Users/runner/.dotnet/tools,/Users/runner/.ghcup/bin,/Users/runner/hostedtoolcache/stack/2.5.1/x64
Executing: /Users/runner/work/1/s/build/tools/dotnet-gitversion -output json -updateassemblyinfo -verbosity Debug
{
'''

**Comment by augustoproiete** [comment]
@nick5454 Are you able to put together an example repo that reproduces this error?

One thing that looks suspicious is that on Windows it doesn't seem to be using the global tool, but instead a version installed via Chocolatey - You might want to double-check if you're running the version of GitVersion that you expect

**Reply by gitfool** [reply]
@augustoproiete that could be related to what I saw on GitHub Actions in https://github.com/cake-build/cake/issues/2743?

**Reply by augustoproiete** [reply]
@gitfool It does look like it could be the exact fix you added for #2743, which made it to Cake v1.0 and would explain @nick5454 seeing this issue on Cake v0.38.5

@nick5454 Are you able to upgrade Cake v1.0 (preferably v1.1 which is the latest, and you won't need `Cake.DotNetTool.Module` anymore)? Looks like you ran into a fixed issue.

**Reply by nick5454** [reply]
@augustoproiete i'll try to upgrade today

**Reply by nick5454** [reply]
@augustoproiete is there a guide to upgrading or I just need to pin to 1.0 and all is good?

**Reply by nils-a** [reply]
@nick5454 have a look at https://cakebuild.net/docs/getting-started/upgrade#cake-0.38.x-to-cake-1.0.0
